crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__parenthesized__tuple.py.snap

```diff
@@ -0,0 +1,399 @@
+                                1,
+                            ),
+                        },
+                    ),
+                },
+            ),
+            Expr(
+                StmtExpr {
+                    range: 201..202,
+                    value: NumberLiteral(
+                        ExprNumberLiteral {
+                            range: 201..202,
+                            value: Int(
+                                2,
+                            ),
+                        },
+                    ),
+                },
+            ),
+            Expr(
+                StmtExpr {
+                    range: 255..267,
+                    value: Tuple(
+                        ExprTuple {
+                            range: 255..267,
+                            elts: [
+                                Name(
+                                    ExprName {
+                                        range: 255..256,
+                                        id: "x",
+                                        ctx: Load,
+                                    },
+                                ),
+                                Name(
+                                    ExprName {
+                                        range: 258..259,
+                                        id: "y",
+                                        ctx: Load,
+                                    },
+                                ),
+                                NumberLiteral(
+                                    ExprNumberLiteral {
+                                        range: 263..264,
+                                        value: Int(
+                                            2,
+                                        ),
+                                    },
+                                ),
+                                Name(
+                                    ExprName {
+                                        range: 266..267,
+                                        id: "z",
+                                        ctx: Load,
+                                    },
+                                ),
+                            ],
+                            ctx: Load,
+                            parenthesized: false,
+                        },
+                    ),
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | # Test cases for tuple expressions where the parser recovers from a syntax error.
+2 | 
+3 | (,)
+  |  ^ Syntax Error: Expected an expression
+4 | 
+5 | (1,,2)
+  |
+
+
+  |
+3 | (,)
+4 | 
+5 | (1,,2)
+  |    ^ Syntax Error: Expected an expression or a ')'
+6 | 
+7 | (1,,)
+  |
+
+
+  |
+5 | (1,,2)
+6 | 
+7 | (1,,)
+  |    ^ Syntax Error: Expected an expression or a ')'
+8 | 
+9 | # Missing comma
+  |
+
+
+   |
+ 9 | # Missing comma
+10 | (1 2)
+   |    ^ Syntax Error: expected Rpar, found Int
+11 | 
+12 | # Dictionary element in a list
```

Comment:
Future: I think that's worth imporving by using the tuple parsing if 
* Not at a `)`
* Not at a `,`
* At the start of an expression